myClass.ts

```diff
@@ -0,0 +1,48 @@
+class Myclass {
+	firstNumber: number = 0;
+	lastNumber: number = 0;
```

Comment:
These two values are not being used in this class... Maybe you forgot to add methods using them? Or maybe they should not be in this class?